test/staking/allocation.test.ts

```diff
@@ -30,16 +31,65 @@ enum AllocationState {
   Closed,
 }
 
+const ABI_LIB_EXPONENTIAL = [
```

Comment:
Could this be imported from hardhat artifacts instead?

Reply:
totally! Tracking here https://github.com/graphprotocol/contracts/issues/887